### DISCUSSION TITLE
Add an old raw input toggle

### DISCUSSION THREAD

**Discussion by Wynfhi** [ppy/osu#26107] [Ideas]
Currently as it stands, my tablet gets royally screwed over when using the current osu!lazer sensitivity (I move my pen and it moves for like .5 seconds and then stops) When I turn off high precision mouse my cursor moves too fast. this most definitely has something to do with the fact my tablet is on relative input not absolute, but I have been using this since day 3 of playing osu! so I don't feel like it would be worth it to change at this point.

**Comment by peppy** [comment]
Check tablet support in game setting? There's a checkbox and settings for tablet now. "high precision" is intended for mouse, not table.
